web/src/engine/websites/LineWebtoon.ts

```diff
@@ -0,0 +1,261 @@
+
+        try {
+            // Process motion webtoon
+            if (document.querySelector('div#ozViewer div.oz-pages')) {
+                let templateURLs = window.__motiontoonViewerState__.motiontoonParam.pathRuleParam;
+                let uri = window.__motiontoonViewerState__.motiontoonParam.viewerOptions.documentURL;
+                let response = await fetch(uri);
+                let data = await response.json();
+                for (let page of data.pages) {
+                    console.log('PAGE:', page.id);
+                    for (let layer of page.layers) {
+                        let layerAsset = layer.asset.split('/');
+                        let layerAssetFile = data.assets[layerAsset[0]][layerAsset[1]];
+                        let layerAssetExtension = layerAssetFile.split('.').pop();
+                        if (layer.type === 'image') {
+                            layer.asset = templateURLs['image'][layerAssetExtension].replace('{=filename}', layerAssetFile);
+                        }
+                        console.log('  LAYER:', layer.type, '=>', layer.asset);
+                        for (let keyframe in layer.effects) {
+                            let effect = layer.effects[keyframe]['sprite'];
+                            if (effect && effect.type === 'sprite') {
+                                let effectAsset = effect.asset.split('/');
+                                let effectAssetFile = data.assets[effectAsset[0]][effectAsset[1]];
+                                let effectAssetExtension = effectAssetFile.split('.').pop();
+                                effect.asset = templateURLs['image'][effectAssetExtension].replace('{=filename}', effectAssetFile);
+                                console.log('    EFFECT:', effect.type, '=>', effect.asset);
+                                for (let index in effect.collection) {
+                                    let collectionAsset = effect.collection[index].split('/');
+                                    let collectionAssetFile = data.assets[collectionAsset[0]][collectionAsset[1]];
+                                    let collectionAssetExtension = collectionAssetFile.split('.').pop();
+                                    effect.collection[index] = templateURLs['image'][collectionAssetExtension].replace('{=filename}', collectionAssetFile);
+                                    console.log('      COLLECTION:', index, '=>', effect.collection[index]);
+                                }
+                            }
+                        }
+                    }
+                }
+                resolve(data.pages);
+            } else {
+            // Process hard-sub webtoon (default)
+                let images = [...document.querySelectorAll('div.viewer div.viewer_lst div.viewer_img img[data-url]')];
+                let links = images.map(element => new URL(element.dataset.url, window.location).href);
+                resolve(links);
```

Comment:
How about removing the `type` parameter from URL to get higher quality?

Reply:
I just saw that its done in LoadImage but i forgot non layered pictures